website/pages/chapters/assessing_performance.js

```diff
@@ -201,29 +265,37 @@ export default function AssessingPerformance() {
         />
 
         <p>
-          Where <IM math={`p(x, y)`} /> is the probability of seeing the
-          specific input/output <IM math={`(x, y)`} />. Notice, this looks a lot
-          like a standard definition of expectation which is precisely what the
-          true error tries to measure! There is just the added complexity of
-          dealing with the <IM math={`(x, y)`} /> pairs. So then selecting the
-          model that generalizes best, would then mean choosing the one with the
+          This definition should be reminiscent of a formula for an expectation
+          (since that is what it is computing), with a few modifications. Now,
```

Comment:
What are the modifications? This is the exact definition of the discrete expectation?

Reply:
I guess my assumption is that they haven't seen an expectation over a discrete distribution, so this formula should look similar to the one they know, but not exactly the same.